lib/autofill.js

```diff
@@ -2,93 +2,118 @@
 
 const Promise = require('bluebird');
 const url = require('url');
+const Inputs = require('./inputs');
```

Comment:
If I were a pedant I would say that `Inputs` exports a function not a class so the variable name shouldn't be capitalised. I don't actually care that much though